### ISSUE TITLE
Task 5 Prob 3 Ellipse Fit Ransac

### ISSUE THREAD

**Issue by MACILLAS** [chulminy/CIVE497-CIVE700#55]
For problem 3 are we suppose to use the matlab ransac function or implement our own like in question 2? 
Also I'm having a really hard-time trying to figure out how to fit an ellipse from points... Can/should we use functions from matlab community like fitellipse.m ?(https://www.mathworks.com/matlabcentral/fileexchange/15125-fitellipse-m) 

Thanks!

**Comment by chulminy** [comment]
No. You should not use the function. Please ask Mansour or Steven. They already solved the problems.
